src/components/Map/Map.tsx

```diff
@@ -78,10 +83,11 @@ const NoscriptMessage = () => (
 
 const Map = () => {
   const { mapLoaded } = useMapStateContext();
+  const { type } = useClimbingFiltersContext();
 
   return (
     <>
-      <BrowserMapDynamic />
+      <BrowserMapDynamic climbingFilters={{ type }} />
```

Comment:
Could you please use the context inside the `BrowserMap` component? 

It is almost a rule of thumb - do not pass props, when you can use a context inside.